src/media/media-upload.dto.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { IsDate, IsString } from 'class-validator';
+
+export class MediaUploadDto {
+  @IsString()
+  link: string;
+
+  @IsString()
+  noteId: string;
+
+  @IsDate()
+  createdAt: Date;
+
+  @IsString()
+  userName: string;
```

Comment:
These should get JSDoc-Strings like the other Dtos…

Reply:
fixed